Readme.md

```diff
@@ -657,6 +658,67 @@ connection.query(
 );
 ```
 
+### Using Template Literals
+
+Template literals can be used with the mysql module. This allows for cleaner, readable code.
+
+Queries that do not require user provided values can be written as simple template literals as shown below:
+
+```js
+var query = `CREATE TABLE Users(
+  EmailAddress VARCHAR(75) NOT NULL UNIQUE,
+  FirstName VARCHAR(30) NOT NULL,
+  LastName VARCHAR(30) NOT NULL,
+  CONSTRAINT PK_User PRIMARY KEY (EmailAddress)
+  )`;
+```
+
+**However**, queries involving user provided values require escaping values you are inserting into the sql or this will result in sql syntax error. See the [escaping section](#escaping-query-values):
+
+```js
+var email = "user provided email value";
+var first_name = "user provided first name value";
+var last_name = "user provided last name value";
+
+var query = `INSERT INTO Users SET
+    EmailAddress = ${mysql.esacpe(email)},
+    FirstName = ${mysql.escape(first_name)},
+    LastName = ${mysql.escape(last_name)},
+    ON DUPLICATE KEY UPDATE
+    FirstName = ${mysql.escape(first_name)},
+    LastName = ${mysql.escape(last_name)}`;
+```
+
+Placeholder syntax can also be used as follows:
+
+```js
+var email = "user provided email value";
+var first_name = "user provided first name value";
+var last_name = "user provided last name value";
+
+var query = `mysql.format(INSERT INTO Users SET
```

Comment:
Sorry I missed this on the last review. Was just going through trying to run the examples real quick. I think the quote on this line is in the wrong spot. I think it's supposed to be after the opening parentheses.